drivers/sensor/bma280/bma280_trigger.c

```diff
@@ -265,6 +277,9 @@ int bma280_init_interrupt(struct device *dev)
 		return -EIO;
 	}
 
+#ifdef CONFIG_BMA280_TRIGGER
```

Comment:
This seems to be unnecessary, as this file is compiled only when `CONFIG_BMA280_TRIGGER` is defined.

Reply:
done